core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java

```diff
@@ -327,37 +324,43 @@ private boolean putInternal(PageId pageId, byte[] page) {
           mMetaStore.addPage(pageId, new PageInfo(pageId, page.length));
         }
       }
-      if (deletePage(victim, victimPageInfo)) {
+      // Regardless enoughSpace, evicting victim to make progress
```

Comment:
Not sure if this comment is accurate, we already removed the victim from the metastore on line 316, so I think we need to go through with the ondisk delete?